x-pack/platform/plugins/shared/integration_assistant/server/graphs/api_analysis/prompts.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ChatPromptTemplate } from '@langchain/core/prompts';
+
+export const SUGGESTED_PATHS_PROMPT = ChatPromptTemplate.fromMessages([
+  [
+    'system',
+    `You are a helpful, expert assistant in REST APIs.
+Here is some context for you to reference for your task, read it carefully as you will get questions about it later:
+<context>
+<path_options>
+{path_options}
+</path_options>
+</context>`,
+  ],
+  [
+    'human',
+    `Review each of the path_options specified as each option represents a REST endpoint path and a short description of that path. Please determine from the provided options
+a list of recommendations for which paths to use to retrieve data relevant to {data_stream_title}. Return at least 1, but up to 4 options in order of best fit. Be sure
+to only respond with exact path from the options provided.
+
+You ALWAYS follow these guidelines when writing your response:
+ <guidelines>
+ - Prioritize bulk api routes over more specialized routes.
+ - Try and return as many options from the provided list as possible, while maintaining preference order.
+ - Your response must only include exact paths specified in the path_options.
+ </guidelines>
+
```

Comment:
We should add a response example like we do in other prompts.
I tried without the sample response and the LLM response is not always in the expected format.
With the `anthropic.claude-3-5-sonnet-20240620-v1:0` connector and it works fine, however `anthropic.claude-3-opus-20240229-v1:0` will fail as it returns a different output.

The error:
<img width="856" alt="Screenshot 2024-12-30 at 16 30 18" src="https://github.com/user-attachments/assets/3fef012a-1224-49a9-ad31-93e443e6d65a" />

Working scenario (LLM returns only the list of paths, or empty list):
<img width="657" alt="Screenshot 2024-12-30 at 16 26 47" src="https://github.com/user-attachments/assets/b1f0080f-63c9-4fdf-ba14-ff3800efbc7e" />

Failing scenario (LLM returns the list + a sentence):
<img width="649" alt="Screenshot 2024-12-30 at 16 26 33" src="https://github.com/user-attachments/assets/7a13ae98-f119-48c8-9ae3-16c6c667105c" />

With adding something like below it fixes the issue:
```
Please respond only with a string array of the suggested paths. Empty array if no paths are relevant.
Example response format:
<example>
A: Please find the suggested paths below:
\`\`\`json
  []
\`\`\`
</example>
`
```